python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -290,6 +293,9 @@ def submit(
     ) -> Iterator[Union[Block, List[BlockMetadata]]]:
         yield from _map_task(fn, ctx, *blocks)
 
+    def __repr__(self):
+        return f"MapWorker({self.src_fn_name})"
```

Comment:
I am actually feeling the `MapWorker` naming might be confusing for user, shall we rename it to be more specific - like `MapBatchesActor`? @clarkzinzow

Reply:
Agreed, but I think that the current `MapWorker(MapBatches(fn_name))` is pretty good!

Reply:
Once https://github.com/ray-project/ray/issues/32922 is implemented, this should allow us to send this actor name to the dash